src/qt/clientmodel.cpp

```diff
@@ -29,6 +29,7 @@
 #include <stdint.h>
 
 #include <QDebug>
+#include <QThread>
```

Comment:
Do NOT backport 17252, please backport 17135 instead! (note that the diff is the same)

Reply:
Renamed commit to reference 17135 instead, force pushed!